### ISSUE TITLE
LocalhostSplitManager always returns null when getting a split by feature name.

### ISSUE THREAD

**Issue by edwardsre** [splitio/java-client#187]
When using a localhost split file and calling `SplitManager.split("my-feature")`, `null` is always returned. This is due to the `containsKey(featureName)` is doing an equals comparison between a `String` (parameter) and a `SplitAndKey` (the map key type) which can never be `true`.

https://github.com/splitio/java-client/blob/42ff3e219d381f49e5fe5734031fec368f2e7d49/client/src/main/java/io/split/client/LocalhostSplitManager.java#L80

Contents of local **split.yaml**
```yaml
- my-feature:
    treatment: "on"
    config: "{\"desc\" : \"this applies only to ON treatment\"}"
```

It seems like the if condition should be using `_splitToTreatmentsMap.contains(featureName)` instead of `_splitAndKeyToTreatmentMap`.

**Comment by chillaq** [comment]
Hi @edwardsre , I will check with the team, keep in mind, when using localhost, only Manager.splitNames() is valid, however, Manager.SplitView will not work since it contain fields that do not exist in localhost mode, like "changeNumber", "TrafficType" and "Killed".

**Comment by edwardsre** [comment]
@chillaq I was expecting the call to return a split view for a specific feature name to return the same data as when I issue a call to splits() which returns all of them. Not sure why the specific one would return null when I can see them all in the list.

**Comment by chillaq** [comment]
Thanks @edwardsre , we are working on a fix.

**Comment by nmayorsplit** [comment]
Hi @edwardsre, I checked and it was fixed in Java SDK 4.1.4  version.

Thanks 
Nadia
